src/common/containers/GameArea.js

```diff
@@ -237,11 +240,17 @@ export class GameArea extends Component {
   constructor(props) {
     super(props);
 
-    if (!props.started) {
-      if (props.history.location.pathname.includes('/tutorial')) {
-        props.onStartTutorial();
+    const { started, history, location, onStartTutorial } = props;
+
+    // If the game hasn't started yet, that means that the player got here
+    // by messing with the URL (rather than by clicking a button in the lobby).
+    // If the URL is '/play/tutorial', just start the tutorial.
+    // Otherwise, return to the lobby because we can't do anything else.
+    if (!started) {
+      if (location.pathname.startsWith(Play.urlForGameMode('tutorial'))) {
+        onStartTutorial();
       } else {
-        props.history.push('/play');
+        history.push(Play.urlForGameMode(null));
```

Comment:
I don't like the passing `null` here as a convention. Maybe two different methods. `urlForTutorialMode` and `urlForPlayMode`?

Reply:
@john-patterson How about now?